src/serlo-editor/plugins/page-layout/editor.tsx

```diff
@@ -40,6 +50,38 @@ export const PageLayoutEditor: React.FunctionComponent<
     </>
   )
 
+  function renderPluginToolbar() {
+    if (!focused) return null
+
+    const layoutStrings = editorStrings.plugins.layout
+
+    return (
+      <PluginToolbar
+        pluginType={EditorPluginType.PageLayout}
+        pluginControls={<PluginDefaultTools pluginId={id} />}
+        pluginSettings={
+          <>
+            <button
+              onClick={() => setShowSettingsModal(true)}
+              className="mr-2 rounded-md border border-gray-500 px-1 text-sm transition-all hover:bg-editor-primary-200 focus-visible:bg-editor-primary-200"
+            >
+              {layoutStrings.chooseLayout} <FaIcon icon={faPencilAlt} />
+            </button>
+            {showSettingsModal ? (
+              <ModalWithCloseButton
+                isOpen={showSettingsModal}
+                onCloseClick={() => setShowSettingsModal(false)}
+                className="!top-1/3 !max-w-xl"
+              >
+                {renderInlineSettings()}
+              </ModalWithCloseButton>
+            ) : null}
+          </>
+        }
+      />
+    )
+  }
+
```

Comment:
code is good. I will refactor it into a `toolbar.tsx`. I think it's a nice way to split the setting into own components.